swe_getSPM.m

```diff
@@ -187,7 +187,9 @@
 %-GUI setup
 %--------------------------------------------------------------------------
 %spm_help('!ContextHelp',mfilename)
-spm('Pointer','Arrow')
+if ~exist('OCTAVE_VERSION','builtin')
```

Comment:
For my edification, what gets screwed up in Octave with `spm('pointer','Arrow')`?

Reply:
I think this must have been a mistake...